build.common.gradle

```diff
@@ -0,0 +1,88 @@
+ext.sonatypeUser = project.hasProperty("sonatypeUser") ? project.sonatypeUser : System.env.SONATYPE_USER as String
```

Comment:
Can simplify these two lines by simply taking the values from the system env variables, we won't be specifying them them in the project build.

Reply:
agree